architecture/service_catalog/aws_broker.adoc

```diff
@@ -0,0 +1,73 @@
+[[admin-guide-get-started-aws]]
+= {product-title} Amazon Web Services Broker
```

Comment:
Maybe "Installing the {product-title} Amazon Web Services Broker"

Reply:
@kalexand-rh I was matching the other titles in the Service Catalog Components book.